x-pack/test/api_integration/apis/ml/data_frame_analytics/get.ts

```diff
@@ -273,5 +273,36 @@ export default ({ getService }: FtrProviderContext) => {
         expect(body).to.have.keys('elements', 'details', 'error');
       });
     });
+
+    describe('GetDataFrameAnalyticsMessages', () => {
+      it('should fetch single analytics job messages by id', async () => {
+        const { body } = await supertest
+          .get(`/api/ml/data_frame/analytics/${jobId}_1/messages`)
+          .auth(USER.ML_VIEWER, ml.securityCommon.getPasswordForUser(USER.ML_VIEWER))
+          .set(COMMON_REQUEST_HEADERS)
+          .expect(200);
+
+          expect(body.length).to.eql(12);
+          expect(body[0].job_id).to.eql(`${jobId}_1`);
+          expect(body[0]).to.have.keys(
+            'job_id',
+            'message',
+            'level',
+            'timestamp',
+            'node_name',
+            'job_type'
+        );
+      });
+
+      it('should not allow to retrieve a job stats for the user without required permissions', async () => {
```

Comment:
Is this a copy and paste from one of the job stats tests?

Reply:
Yep - accidentally left in - fixed in 49371743f19221b4c20d62d87e9227545366a280